Public/Src/Sandbox/MacOs/Sandbox/Src/Kauth/TrustedBsdHandler.cpp

```diff
@@ -94,8 +99,8 @@ int TrustedBsdHandler::HandleVnodeWrite(vnode_t vnode)
 
 // TODO: We could take advantage of knowing what's on critical path, and not slow down those processes
 //       This information could be conveyed via the FileAccessManifest
-void TrustedBsdHandler::HandleProcessWantsToFork(const pid_t parentProcessPid)
-{
+void TrustedBsdHandler::HandleProcessWantsToFork(const pid_t parentProcessPid){
```

Comment:
Nit: but curly braces back to line below ;)

Reply:
this must have been some Xcode auto formatting business